lib/core/Canvas.js

```diff
@@ -347,9 +345,23 @@ Canvas.prototype._clear = function() {
 
 /**
 * Sets focus on the canvas SVG element.
+*
+* @param {boolean} [onlyFromBody=true] If true, will only focus canvas if
+*     current active element is `document.body`. True by default.
+*/
+Canvas.prototype.focus = function(onlyFromBody = true) {
```

Comment:
I have split the focus logic into `focus` and `restoreFocus` functions.

Since `restoreFocus` seems to be the default to use externally, I think it makes sense to keep both public.